### PR TITLE
Identity Framework Added.

### PR DESCRIPTION
**Pull request by hard-rox**
I added Identity in the project. But it is not functional. Please take a look.

### CODE REVIEW

**Comment by kajal-alpha** [comment]
ViewModel Should be add in in AssetManager.Web. Please modify.

File: src/AssetManager.Web/Startup.cs [hunk 1]
```diff
@@ -21,6 +23,21 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
+        public void ConfigureServices(IServiceCollection services)
```
Comment (ratanparai) [inline src/AssetManager.Web/Startup.cs]: There is big problem with your code here. There was already `ConfigureServices` method that get called by `Configure[environment]Services` , example `ConfigureDevelopmentServices` . Don't blindly copy paste any codes. 

> Continue next comment

File: src/AssetManager.Web/Startup.cs [hunk 2]
```diff
@@ -21,6 +23,21 @@ public Startup(IConfiguration configuration)
...
+                .AddDefaultTokenProviders();
+
+            // Add application services.
+            //services.AddTransient<IEmailSender, EmailSender>();
+
+            services.AddMvc();
+        }
+
         // This method gets called when Development Environment is used
         // Use this method to set Development services, like Development database
```
Comment (ratanparai) [inline src/AssetManager.Web/Startup.cs]: This method get invoked when we run the app from visual studio (Development Environment) 

We are using the same configuration of `Testing Environment` . 

Also check that we are configuring **Database** inside those **Environment Specific** methods. Not **Generic** `ConfigureServices` method.

File: src/AssetManager.Web/Startup.cs [hunk 3]
```diff
@@ -55,13 +72,13 @@ public void ConfigureProductionServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddScoped(typeof(IAsyncRepository<>), typeof(EfRepository<>));
+        //public void ConfigureServices(IServiceCollection services)
```
Comment (ratanparai) [inline src/AssetManager.Web/Startup.cs]: Always remember,
> Never comment out any codes
It generate code smells. Everyone eventually forget why the code is commented out after few weeks.

File: src/AssetManager.Web/Startup.cs [hunk 4]
```diff
@@ -55,13 +72,13 @@ public void ConfigureProductionServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddScoped(typeof(IAsyncRepository<>), typeof(EfRepository<>));
```
Comment (ratanparai) [inline src/AssetManager.Web/Startup.cs]: This line is registering `EfRepository` to `IOC` container. This is required for controller to get repository via `constructor` injection. If you remove this line, your code will not run.

**Comment by ratanparai** [comment]
@kajal-alpha : Please stop merging changes from master after every PR accept. You should never do that. This is not your job.

**Comment by kajal-alpha** [comment]
understand ... @ratanparai bro... 👍

**Pull request closed without merging**